build.gradle

```diff
@@ -77,10 +77,12 @@ task clean(type: Delete) {
   delete rootProject.buildDir
 }
 
-def BpkVersion = '33.0.1'
+def BpkAndroidVersion = '33.0.1'
```

Comment:
Any reason to keep these here, rather than delegate versioning to the modules themselves?